### PR TITLE
Release 0.7.0

### PR DESCRIPTION
**Pull request by huard**
## Overview

This PR updates the CHANGES and bumps version to 0.7.0

### CODE REVIEW

**Comment by huard** [comment]
@tlvu There doesn't seem to be a notebook environment.yml file to indicate dependencies.

**Comment by tlvu** [comment]
> @tlvu There doesn't seem to be a notebook environment.yml file to indicate dependencies.

Should be from the same "dev" env for the bird.

**Comment by tlvu** [comment]
> > @tlvu There doesn't seem to be a notebook environment.yml file to indicate dependencies.
> 
> Should be from the same "dev" env for the bird.

Indeed, same "dev" env since it assume a running local Finch see https://github.com/bird-house/finch/blob/babd6fafe12b925c8123f6c873d3629d939aa41c/Makefile#L5 and https://github.com/bird-house/finch/blob/babd6fafe12b925c8123f6c873d3629d939aa41c/Makefile#L139

File: docs/source/notebooks/finch-usage.ipynb [hunk 1]
```diff
@@ -140,77 +144,991 @@
      "output_type": "stream",
      "text": [
       "Process status:  ProcessSucceeded\n",
-      "Link to process output:  https://pavics.ouranos.ca/wpsoutputs/6098a61c-7ae6-11eb-9131-9c305b4af68f/frost-days_SRES-A2-experiment_20460101-20650101.nc\n"
+      "Link to process output:  http://localhost:5000/outputs/62b3ef56-859a-11eb-a5eb-a44cc85b43bf/frost-days_SRES-A2-experiment_20460101-20650101.nc\n"
```
Comment (tlvu) [inline docs/source/notebooks/finch-usage.ipynb:147]: Did you run `refresh-notebooks`?  `refresh-notebooks` should automatically replace `localhost:5000` with `pavics.ouranos.ca`

**Review by tlvu — CHANGES_REQUESTED** [review]
Everything looks fine, except the lax part and `refresh-notebooks`.

File: docs/source/notebooks/finch-usage.ipynb (hunk 1 above)
Comment (huard) [reply docs/source/notebooks/finch-usage.ipynb]: Ok, but it also seems to strip the html output. After refresh, I'm back with text output... weird.

**Comment by huard** [comment]
@tlvu ok comme ça ?

File: docs/source/notebooks/dap_subset.ipynb [hunk 2]
```diff
@@ -497,20 +497,21 @@
...
       ]
      },
-     "execution_count": 12,
+     "execution_count": 2,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
+    "#NBVAL_IGNORE_OUTPUT\n",
```
Comment (tlvu) [inline docs/source/notebooks/dap_subset.ipynb:506]: In the nbval doc, there is a space between `#` and `NBVAL_IGNORE_OUTPUT`.  Not sure if it has to be exactly like that or not.  If you run `make test-notebooks`, everything pass, even without lax?  If yes then all good.

File: docs/source/notebooks/dap_subset.ipynb [hunk 3]
```diff
@@ -946,7 +948,7 @@
...
...r-var-dtype'>float32</div><div class='xr-var-preview xr-preview'>...</div><input id='attrs-cbcb535a-0d95-4040-912a-9092a9a9de11' class='xr-var-attrs-in' type='checkbox' ><label for='attrs-cbcb535a-0d95-4040-912a-9092a9a9de11' title='Show/Hide attributes'><svg class='icon xr-icon-file-text2'><use xlink:href='#icon-file-text2'></use></svg></label><input id='data-458c54ce-7904-4afe-8a4e-d822ad4b6d22' class='xr-var-data-in' type='checkbox'><label for='data-458c54ce-7904-4afe-8a4e-d822ad4b6d22' title='Show/Hide data repr'><svg class='icon xr-icon-database'><use xlink:href='#icon-database'></use></svg></label><div class='xr-var-attrs'><dl class='xr-attrs'><dt><span>original_name :</span></dt><dd>APRL+APRC</dd><dt><span>long_name :</span></dt><dd>Precipitation</dd><dt><span>standard_name :</span></dt><dd>precipitation_flux</dd><dt><span>cell_methods :</span></dt><dd>time: mean (interval: 30 minutes)</dd><dt><span>units :</span></dt><dd>kg m-2 s-1</dd><dt><span>history :</span></dt><dd>pr=max(0,pr) applied to raw data;</dd><dt><span>_ChunkSizes :</span></dt><dd>[7200    6    7]</dd></dl></div><div class='xr-var-data'><pre>array([[[3.415346e-05, 2.171099e-05, 1.613796e-05, 4.132092e-05,\n",
```
Comment (tlvu) [inline docs/source/notebooks/dap_subset.ipynb:951]: I still see html here, I am a bit surprised you say xarray rendering reverted back to text.

**Review by tlvu — APPROVED** [review]
Yup, good for me.  If you have not tried a `make test-notebooks` after your `make refresh-notebooks` try it and make sure it passes.

File: docs/source/notebooks/dap_subset.ipynb (hunk 2 above)
Comment (huard) [reply docs/source/notebooks/dap_subset.ipynb:506]: Yes.

File: docs/source/notebooks/dap_subset.ipynb (hunk 3 above)
Comment (huard) [reply docs/source/notebooks/dap_subset.ipynb:951]: When I open the notebook, I get text output, although when I ran the notebook manually, I got html...

**Pull request merged**